cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/requestrate/RequestRateConfiguration.java

```diff
@@ -13,11 +16,23 @@
 public class RequestRateConfiguration {
 
     /**
+     * Default is to report all requests.
+     */
+    public static final String ALL_REQUESTS_TO_REPORT = "ALL";
```

Comment:
This has the same value as CL.ALL and can lead to confusion in implementation. Consider enabling configuration via ConsistencyLeve:StatementType and then *:* is all and *:SELECT reports all CL for select statements and ONE:* reports CL ONE and all statement type..